projects/rocprofiler-compute/src/utils/utils.py

```
@@ -1011,72 +1019,126 @@ def pc_sampling_prof(
         console_error("PC sampling failed.")
 
 
-def process_rocprofv3_output(
-    rocprof_output: str, workload_dir: str, is_timestamps: bool
-) -> list[str]:
+def process_rocprofv3_output(workload_dir: str, using_native_tool: bool) -> list[str]:
     """
-    rocprofv3 specific output processing.
-    takes care of json or csv formats, for csv format,
-    additional processing is performed.
+    rocprofv3 specific output processing for csv format.
     """
     results_files_csv: list[str] = []
 
-    if rocprof_output == "json":
-        results_files_json = glob.glob(f"{workload_dir}/out/pmc_1/*/*.json")
+    # Convert native counter collection csv to rocprofv3 counter collection csv format
```

Could explain a bit more why do we need to do conversion here?
In case it might fail, should we try catch here?

---

We already have logic to process rocprofiler-sdk tool counter collection csv to pmc_perf.csv
When using native tool for counter collection, the counter collection csv has to be converted to rocprofiler-sdk tool format to leverage this logic.

If it fails, then we should error out, we cannot silently catch it since that will result in no counter collection data!

---

I'm not saying "silently catch". I mean catch the error properly and error out